loops.lab.js

```diff
@@ -0,0 +1,186 @@
+// // 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
```

Comment:
Overall good demonstration of use and implementation of loops. I only had minor feedbacks and did not see any major errors. Good job, keep up the good work, and keep doing bonuses questions!